src/PhpSpreadsheet/Reader/Ods/PageSettings.php

```diff
@@ -54,10 +54,10 @@ private function readPageSettingStyles(DOMDocument $styleDom): void
             $marginBottom = $pageLayoutProperties->getAttributeNS($this->stylesFo, 'margin-bottom');
             $header = $styleSet->getElementsByTagNameNS($this->stylesNs, 'header-style')[0];
             $headerProperties = $header->getElementsByTagNameNS($this->stylesNs, 'header-footer-properties')[0];
-            $marginHeader = $headerProperties->getAttributeNS($this->stylesFo, 'min-height');
+            $marginHeader = isset($headerProperties) ? $headerProperties->getAttributeNS($this->stylesFo, 'min-height') : null;
```

Comment:
Looking forward to the day when we have PHP8 as a minimum version, and can use `?->`